[2025-01-07T15:34:22.232+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_teams_data.truncate_nba_teams manual__2025-01-07T15:34:20.628487+00:00 [queued]>
[2025-01-07T15:34:22.246+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_teams_data.truncate_nba_teams manual__2025-01-07T15:34:20.628487+00:00 [queued]>
[2025-01-07T15:34:22.246+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-01-07T15:34:22.263+0000] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): truncate_nba_teams> on 2025-01-07 15:34:20.628487+00:00
[2025-01-07T15:34:22.269+0000] {standard_task_runner.py:57} INFO - Started process 2495 to run task
[2025-01-07T15:34:22.275+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fetch_teams_data', 'truncate_nba_teams', 'manual__2025-01-07T15:34:20.628487+00:00', '--job-id', '470', '--raw', '--subdir', 'DAGS_FOLDER/process_teams_dag.py', '--cfg-path', '/tmp/tmpnjtuzjcx']
[2025-01-07T15:34:22.281+0000] {standard_task_runner.py:85} INFO - Job 470: Subtask truncate_nba_teams
[2025-01-07T15:34:22.367+0000] {task_command.py:410} INFO - Running <TaskInstance: fetch_teams_data.truncate_nba_teams manual__2025-01-07T15:34:20.628487+00:00 [running]> on host 27ff283f09fe
[2025-01-07T15:34:22.861+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_teams_data' AIRFLOW_CTX_TASK_ID='truncate_nba_teams' AIRFLOW_CTX_EXECUTION_DATE='2025-01-07T15:34:20.628487+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-07T15:34:20.628487+00:00'
[2025-01-07T15:34:22.866+0000] {sql.py:265} INFO - Executing: TRUNCATE TABLE nba_teams;
[2025-01-07T15:34:22.875+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-01-07T15:34:22.938+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-01-07T15:34:22.941+0000] {sql.py:375} INFO - Running statement: TRUNCATE TABLE nba_teams;, parameters: None
[2025-01-07T15:34:22.942+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 277, in execute
    **extra_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 349, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 380, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedTable: relation "nba_teams" does not exist

[2025-01-07T15:34:22.949+0000] {taskinstance.py:1350} INFO - Marking task as UP_FOR_RETRY. dag_id=fetch_teams_data, task_id=truncate_nba_teams, execution_date=20250107T153420, start_date=20250107T153422, end_date=20250107T153422
[2025-01-07T15:34:22.957+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 470 for task truncate_nba_teams (relation "nba_teams" does not exist
; 2495)
[2025-01-07T15:34:23.002+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2025-01-07T15:34:23.039+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
